docs/vspec2x_arch.md

```diff
@@ -22,6 +19,28 @@ This will result in that e.g. `Vehicle.Cabin.Door` is expanded to the following
 For some exporters expansion can be suppressed by using the `--no_expand` option.
 Then instance information will be represented by other means in the resulting output.
 
+## Deletion / Node removal
+
+Nodes can be removed from the tree by using the `delete` keyword in the overlay.
```

Comment:
There is a lot of variation in how node elements (type, datatype, min, max, etc.) are called. Various terms are used, such as "attribute" (confusing, since some nodes have a type of "attribute"), member, element, keyword, etc.

Refer to https://github.com/COVESA/vehicle_signal_specification/issues/733 for a description of the issue.

Given the current recommendation (yet to be approved) is to use the term "element", I would suggest that you use this for now at least.